networking_aci/plugins/ml2/drivers/mech_aci/driver.py

```diff
@@ -44,6 +45,10 @@ def __init__(self):
         self.rpc_notifier = rpc_api.ACIRpcClientAPI(self.context)
         self.start_rpc_listeners()
         self.trunk_driver = ACITrunkDriver.create()
+        self.vif_details = {
+            portbindings.CAP_PORT_FILTER: False,
```

Comment:
The commit message only mentions l2 connectivity. What does `CAP_PORT_FILTER` do for us? From neutron-lib it reads a bit like it is no longer used?

https://github.com/sapcc/neutron-lib/blob/stable/ussuri-m3/neutron_lib/api/definitions/portbindings.py#L39-L41

Reply:
Good point, let's remove it